docs/community_tools.md

```diff
@@ -1,8 +1,11 @@
 # Community-built tools around Jujutsu
 
-**Important:** Many of these are not complete yet, just like Jujutsu itself.
-But they already simplify many workflows and can improve your experience.
+Many of these tools are not complete yet, just like Jujutsu itself. But they
+already simplify many workflows and can improve your experience.
 
+!!! warning
+    The listed tools are community‑maintained; Jujutsu does not review, endorse,
```

Comment:
nit: "the Jujutsu projects' contributors do not..."

Reply:
Not a lawyer. But I guess in that case "the Jujutsu project" and "the Jujutsu project's contributors" are really just the same thing. I'll change it to the former but we can give @thoughtpolice a chance to object.

Reply:
SGTM 👍🏻